test/runtests.jl

```diff
@@ -15,1923 +15,1687 @@ end
-end
-test_trsv!(Float32)
-test_trsv!(Float64)
-test_trsv!(Complex64)
-test_trsv!(Complex128)
-
-function test_trsv(elty)
-    # generate triangular matrix
-    A = rand(elty,m,m)
-    A = triu(A)
-    # generate vector
-    x = rand(elty,m)
-    # move to device
-    d_A = CudaArray(A)
-    d_x = CudaArray(x)
-    # execute trsv!
-    d_y = CUBLAS.trsv('U','N','N',d_A,d_x)
-    y = A\x
-    # compare
-    h_y = to_host(d_y)
-    @test_approx_eq(y,h_y)
-end
-test_trsv(Float32)
-test_trsv(Float64)
-test_trsv(Complex64)
-test_trsv(Complex128)
-
-#############
-# test ger! #
-#############
-
-function test_ger!(elty)
-    # construct matrix and vectors
-    A = rand(elty,m,n)
-    x = rand(elty,m)
-    y = rand(elty,n)
-    alpha = convert(elty,2)
-    # move to device
-    d_A = CudaArray(A)
-    d_x = CudaArray(x)
-    d_y = CudaArray(y)
-    # perform rank one update
-    CUBLAS.ger!(alpha,d_x,d_y,d_A)
-    A = (alpha*x)*y' + A
-    # move to host and compare
-    h_A = to_host(d_A)
-    @test_approx_eq(A,h_A)
-end
-test_ger!(Float32)
-test_ger!(Float64)
-test_ger!(Complex64)
-test_ger!(Complex128)
-
-#############
-# test syr! #
-#############
-
-function test_syr!(elty)
-    # construct matrix and vector
-    A = rand(elty,m,m)
-    A = A + A.'
-    x = rand(elty,m)
-    alpha = convert(elty,2)
-    # move to device
-    d_A = CudaArray(A)
-    d_x = CudaArray(x)
-    # perform rank one update
-    CUBLAS.syr!('U',alpha,d_x,d_A)
-    A = (alpha*x)*x.' + A
-    # move to host and compare upper triangles
-    h_A = to_host(d_A)
-    A = triu(A)
-    h_A = triu(h_A)
-    @test_approx_eq(A,h_A)
-end
-test_syr!(Float32)
-test_syr!(Float64)
-test_syr!(Complex64)
-test_syr!(Complex128)
-
-#############
-# test her! #
-#############
-
-function test_her!(elty)
-    local m = 2
-    # construct matrix and vector
-    A = rand(elty,m,m)
-    A = A + A'
-    x = rand(elty,m)
-    alpha = convert(elty,2)
-    # move to device
-    d_A = CudaArray(A)
-    d_x = CudaArray(x)
-    # perform rank one update
-    CUBLAS.her!('U',alpha,d_x,d_A)
-    A = (alpha*x)*x' + A
-    # move to host and compare upper triangles
-    h_A = to_host(d_A)
-    A = triu(A)
-    h_A = triu(h_A)
-    @test_approx_eq(A,h_A)
-end
-test_her!(Complex64)
-test_her!(Complex128)
-
-
-##############
-# test her2! #
-##############
-
-function test_her2!(elty)
-    local m = 2
-    # construct matrix and vector
-    A = rand(elty,m,m)
-    A = A + A'
-    x = rand(elty,m)
-    y = rand(elty,m)
-    alpha = convert(elty,2)
-    # move to device
-    d_A = CudaArray(A)
-    d_x = CudaArray(x)
-    d_y = CudaArray(y)
-    # perform rank one update
-    CUBLAS.her2!('U',alpha,d_x,d_y,d_A)
-    A = (alpha*x)*y' + y*(alpha*x)' + A
-    # move to host and compare upper triangles
-    h_A = to_host(d_A)
-    A = triu(A)
-    h_A = triu(h_A)
-    @test_approx_eq(A,h_A)
-end
-test_her2!(Complex64)
-test_her2!(Complex128)
-
-##############
-# test gemm! #
-##############
-
-function test_gemm!(elty)
```

Comment:
ah excellent, I just searched for gemm! in the diff and somehow nothing turned up ;)